libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1453,6 +1453,27 @@ void AP_Arming::Log_Write_Disarm(const AP_Arming::Method method)
     };
     AP::logger().WriteCriticalBlock(&pkt, sizeof(pkt));
     AP::logger().Write_Event(LogEvent::DISARMED);
+
```

Comment:
This shouldn't be in `Log_Write_Disarm`.  There's only one caller to this function - it should go next to that call.  We *may* want a new `check_forced_logging()` method or something.